src/resources/lib/abstract-resource.js

```diff
@@ -244,6 +244,15 @@ module.exports = class AbstractResource extends Restypie.Resources.AbstractCoreR
     if (!Utils.isValidNumber(this.defaultLimit)) throw new TypeError('Property `defaultLimit` should be a number');
     if (!Utils.isValidNumber(this.maxLimit)) throw new TypeError('Property `maxLimit` should be a number');
     
+    const schema = this.schema;
+
+    if (this.defaultSelect) {
+      this.defaultSelect.forEach(function (keyItem) {
```

Comment:
I'm trying to use arrow functions everywhere I develop now : `this.defaultSelect.forEach(function (keyItem) {` => `this.defaultSelect.forEach(keyItem => {`